scripts/init_install.sh

```diff
@@ -128,7 +120,7 @@ source payload/txsim.sh
 HOSTNAME=$(hostname)
 
 # Base command
-COMMAND="celestia-appd start --force-no-bbr"
+COMMAND="celestia-appd start"
```

Comment:
since we're setting bbr in all cases, this could serve as a line of defense in case the above BBR configuration wasn't successful